willow/plugins/wand.py

```diff
@@ -183,6 +184,22 @@ def save_as_webp(self, f, quality=80, lossless=False):
 
         return WebPImageFile(f)
 
+    @Image.operation
+    def save_as_avif(self, f, quality=80, lossless=False):
+        with self.image.convert("avif") as converted:
+            converted.compression_quality = quality
+            if lossless:
+                library = _wand_api().library
+                library.MagickSetOption.argtypes = [c_void_p, c_char_p, c_char_p]
+                library.MagickSetOption(
+                    converted.wand,
+                    b"avif:lossless",
```

Comment:
@salty-ivy where did you find this? It would be great to add inline comments for this type of options so we don't scratch our heads 6 months down the line.

As far as I can from https://github.com/ImageMagick/ImageMagick/discussions/3076, it is either 100% quality, or we need to pass `heic:lossless=true` (supported since version 7.1.0-44 or 45)

Reply:
I think my implementation and already existing `save_as_webp` both fails to save image in lossless form.

Reply:
Done just need to take of these lossless test fails.